packages/app-next/app-config.yaml

```diff
@@ -20,12 +20,12 @@ app:
     - entity-card:catalog-graph/relations:
         config:
           height: 300
-    - entity-card:azure-devops/readme
+    # - entity-card:azure-devops/readme
```

Comment:
Disabling these `azure-devops` cards and contents because they throw the following error on the entity page (@awanlin will take a look):

![image](https://github.com/backstage/backstage/assets/6290749/d0798243-9459-4819-8298-f30abdcc6004)